bucket_01/fltk/specification

```diff
@@ -31,6 +31,6 @@ MUST_CONFIGURE=		gnu
 CONFIGURE_ARGS=		--enable-shared --docdir=/raven/share/examples/fltk
 
 post-stage:
-	mkdir -p /construction/fltk/stage/raven/share/examples/fltk
-	mv /construction/fltk/stage/raven/share/doc/fltk/examples/* /construction/fltk/stage/raven/share/examples/fltk
-	rm -r  /construction/fltk/stage/raven/share/doc
+	${MKDIR} -p /construction/fltk/stage/raven/share/examples/fltk
```

Comment:
MKDIR includes "-p", so you can remove "-p" from this line.
FYI, RM includes "-f".